R/utils.R

```diff
@@ -5,4 +5,122 @@ get_file_type <- function(tracks){
   # character vector:
   #  - .bam$
   #  - .bw$
-}
\ No newline at end of file
+}
+
+
+#check that seqlevelsStyle of x and y match
+#' @noRd
+seqlevels_match <- function(x,y) {
+  
+	if(seqlevelsStyle(x) == seqlevelsStyle(y)){
+		return(TRUE)
+	} else{
+		return(FALSE)		
+	}
+}
+
+
+##this is crude and could be improved
+#check that all the seqlevelsStyles found in X are also found in y
+#' @noRd
+seqlevels_style <- function(x,y) {
+  x_styles <- lapply(seqlevels(x), function(a) seqlevelsStyle(a))
+  y_styles <- lapply(seqlevels(y), function(a) seqlevelsStyle(a))
+  if(!all(x_styles %in% y_styles)){
```

Comment:
Similar comment as above, you can just use the logic in the `if` check as the return value since it's already boolean.